uint64 timestamp				# time since system start (microseconds)

uint8 stage					    # stage of the UAV
uint8[20] sta_msg               # message of the stage